base/Dockerfile

```diff
@@ -9,9 +9,5 @@ RUN conda install --yes --freeze-installed \
       ca-policy-lcg \
       xrootd==5.0.3 \
       coffea==0.6.45 \
-      && conda clean -tipsy \
```

Comment:
I would leave this one!

Reply:
I checked with this line and without, size doesn't change...I think the main magic is happening here `conda build purge-all` -which removes all locally built packages as well as in `rm -rf /opt/conda/pkgs`

Reply:
aha ok, so it is maybe better then

Reply:
@nsmith- I think I found better combination with:
```
      && conda clean --all -f -y \
      && conda build purge-all
```